# Lipedema
## QC
Quality control for the raw reads were performed to access the sequence quality, GC content,
the presence of adaptors and the read lengt
Please reffer the script qc.sh
## Mapping 
with bwa mem
 variant calling using gatk or even better Freebayes, 
 variant annotation with SnpEff, 
 script for matching to gnomad data.

rna seq analysis scripts
mapping with star
htseq for gene counts
potentially strelka for the tpms
DESeq/or what Linnaeus used, go enrichment
Leafcutter for the splicing
mutect for somatic variant calling.

the 837 genes we identified to be DE in lipedema
